audit-ci.jsonc

```diff
@@ -4,6 +4,8 @@
 	"low": true,
 	"allowlist": [
 		// Taffdb is only used during jsdoc build and contains only data for jsdoc generation. After jsdoc build finished, db is destroyed
-		"GHSA-mxhp-79qh-mcx6"
+		"GHSA-mxhp-79qh-mcx6",
+		// http-cache-semantics is only used in the development setup. No production impact.
```

Comment:
can you add more details to that?
I haven't looked into it but it is used by `pacote`, which is not a devDependency.